### PR TITLE
Translate `Introduction to HTML and CSS`

### PR DESCRIPTION
**Pull request by CoderMungan**
Orginal File: [intro-to-html-css](https://github.com/TheOdinProject/curriculum/blob/main/foundations/html_css/html-foundations/intro-to-html-css.md)

### CODE REVIEW

File: temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
+Birçok harika kaynak, HTML ve CSS'yi _programlama dilleri_ olarak sürekli bahseder, ancak teknik açıdan yaklaşmak istiyorsanız onları bu şekilde etiketlemek pek doğru değildir. Çünkü yalnızca bilgi sunma ile ilgilidirler. Herhangi bir mantığı programlamak için kullanılmazlar. Bir sonraki bölümde öğreneceğiniz JavaScript, bir programlama dilidir çünkü internet sitelerinin bir şeyler yapmasını sağlamak için kullanılır. Ancak sadece HTML ve CSS ile yapabileceğiniz pek çok şey var ve kesinlikle ikisine de ihtiyacınız olacak. Takip ettiğiniz dersler, müfredatımızda ilerledikçe JavaScript içeriğinde başarılı olmanızı sağlayacak araçları size sunmaktadır.
```
Comment (huseyingulec) [inline temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md:15]: etiketlemek pek doğru değildir, çünkü yalnızca bilgi sunma ile ilgilidirler.

Comment (hueypov) [inline temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md]:
```suggestion
Birçok harika kaynak, HTML ve CSS'yi _programlama dilleri_ olarak sürekli bahseder, ancak teknik açıdan yaklaşmak istiyorsanız onları bu şekilde etiketlemek pek doğru değildir, çünkü yalnızca bilgi sunma ile ilgilidirler. Herhangi bir mantığı programlamak için kullanılmazlar. Bir sonraki bölümde öğreneceğiniz JavaScript, bir programlama dilidir çünkü internet sitelerinin bir şeyler yapmasını sağlamak için kullanılır. Ancak sadece HTML ve CSS ile yapabileceğiniz pek çok şey var ve kesinlikle ikisine de ihtiyacınız olacak. Bundan sonraki dersler, müfredatımıza devam ederken JavaScript içeriğine geldiğinizde başarılı olmanız için gereken araçları sağlamaya odaklanmaktadır!
```

Comment (CoderMungan) [reply temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md]: [ef20af8](https://github.com/kamp-us/turkce-odin-project/pull/54/commits/ef20af861521c4e43a2b9fe3b8cf9c30e5eacfaa) Done!

File: temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md [hunk 2]
```diff
@@ -1,40 +1,40 @@
...
 
-### Assignment
+### Ödev
 
 <div class="lesson-content__panel" markdown="1">
 
-1. Read [HTML vs CSS vs JavaScript](https://brytdesigns.com/html-css-javascript-whats-the-difference/). It's a quick overview of the relationship between HTML, CSS, and JavaScript.
+1. Lütfen Okuyunuz [HTML ile CSS ile de Javascript Arasındaki Farklar](https://brytdesigns.com/html-css-javascript-whats-the-difference/). HTML, CSS ve JavaScript arasındaki ilişkinin kısa bir genel bakışıdır.
```
Comment (hueypov) [inline temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md]:
```suggestion
1. [HTML vs CSS vs JavaScript](https://brytdesigns.com/html-css-javascript-whats-the-difference/) adlı ingilizce makaleyi okuyun. Bu, HTML, CSS ve JavaScript arasındaki ilişkinin kısa bir genel bakışıdır.
```

File: temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md [hunk 3]
```diff
@@ -1,40 +1,40 @@
...
 
 </div>
 
-### Knowledge check
+### Bilgi Ölçme Bakış
```
Comment (hueypov) [inline temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md]:
```suggestion
### Bilgi Ölçme
```

File: temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md [hunk 4]
```diff
@@ -1,40 +1,40 @@
...
 
-This section contains questions for you to check your understanding of this lesson on your own. If you’re having trouble answering a question, click it and review the material it links to.
+Bu bölümde, bu dersi anladığınıza dair, kendinizi kontrol etmeniz için sorular bulunmaktadır. Bir soruyu yanıtlamakta sorun yaşıyorsanız, soruya tıklayın ve bağlantı verdiği materyali inceleyin.
```
Comment (hueypov) [inline temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md]:
```suggestion
Bu bölüm, bu dersi kendi kendinize anlayıp anlamadığınızı kontrol etmeniz için sorular içermektedir. Bir soruyu yanıtlamakta zorlanıyorsanız, soruya tıklayın ve bağlantılı olduğu materyali gözden geçirin.
```

File: temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md (hunk 2 above)
Comment (CoderMungan) [reply temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md]: [3b14e79](https://github.com/kamp-us/turkce-odin-project/pull/54/commits/3b14e79cee29908931c42f9b8f432db9a4761f5c) done!

File: temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md [hunk 5]
```diff
@@ -1,40 +1,40 @@
...
 
-- [What do HTML and CSS stand for?](https://brytdesigns.com/html-css-javascript-whats-the-difference/#What_is_HTML)
-- [Between HTML and CSS, which would you use for putting paragraphs of text on a webpage?](#html-and-css)
-- [Between HTML and CSS, which would you use for changing the font and background color of a button?](#html-and-css)
-- [What is the difference between HTML, CSS and JavaScript?](https://brytdesigns.com/html-css-javascript-whats-the-difference/)
+- [HTML ve CSS ne anlamına geliyor?](https://brytdesigns.com/html-css-javascript-whats-the-difference/#What_is_HTML)
```
Comment (hueypov) [inline temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md]:
```suggestion
- [HTML ve CSS ne anlamına geliyor? adlı ingilizce makale](https://brytdesigns.com/html-css-javascript-whats-the-difference/#What_is_HTML)
```

File: temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md (hunk 4 above)
Comment (CoderMungan) [reply temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md]: [064eab4](https://github.com/kamp-us/turkce-odin-project/pull/54/commits/064eab4f649ae3c1be43ac110737eb6c946efafa) Done!

File: temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md (hunk 3 above)
Comment (CoderMungan) [reply temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md]: [064eab4](https://github.com/kamp-us/turkce-odin-project/pull/54/commits/064eab4f649ae3c1be43ac110737eb6c946efafa) Done!

File: temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md (hunk 5 above)
Comment (CoderMungan) [reply temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md]: [81602bc](https://github.com/kamp-us/turkce-odin-project/pull/54/commits/81602bc297e5bc42208eeb73fe486927e17c6405) Done!

File: temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md [hunk 6]
```diff
@@ -1,40 +1,40 @@
...
+- [HTML, CSS ve JavaScript arasındaki farklar nelerdir?](https://brytdesigns.com/html-css-javascript-whats-the-difference/)
 
-### Additional resources
+### Ek Kaynaklar
 
-This section contains helpful links to related content. It isn’t required, so consider it supplemental.
+Bu alanda içerikle alakalı faydalı linkler bulunmaktadır. Zorunlu değildir, ek olarak düşünülmelidir.
 
-- [This FreeCodeCamp article](https://www.freecodecamp.org/news/html-css-and-javascript-explained-for-beginners/) goes into a little more depth than the assigned one. It covers things we'll be teaching explicitly in later lessons though, so don't worry about memorizing any of the details.
+- [FreeCodeCamp Makalesi](https://www.freecodecamp.org/news/html-css-and-javascript-explained-for-beginners/) belirlenenden biraz daha derine iner. Ancak daha sonraki derslerde açık bir şekilde öğreteceğimiz şeyleri kapsıyor. Bu nedenle ayrıntıları ezberleme konusunda endişelenmeyin.
```
Comment (hueypov) [inline temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md]:
```suggestion
- Bu ingilizce [FreeCodeCamp Makalesi](https://www.freecodecamp.org/news/html-css-and-javascript-explained-for-beginners/) ödev olarak verilen makaleden biraz daha derine iniyor. Daha sonraki derslerde açık bir şekilde öğreteceğimiz şeyleri kapsıyor, bu nedenle herhangi bir ayrıntıyı ezberleme konusunda endişelenmeyin.
```

Comment (CoderMungan) [reply temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md]: [a31bbb5](https://github.com/kamp-us/turkce-odin-project/pull/54/commits/a31bbb51d608f3497d1df36ad1481bff90cb54ff) done!

File: temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md [hunk 7]
```diff
@@ -1,40 +1,40 @@
...
 
-- Bookmark [DevDocs.io](https://devdocs.io). Read the “Welcome” message. Massive API documentation collection that even works offline. An essential collection of reference material for everything covered and more. (Maintained by [FreeCodeCamp](https://freecodecamp.org))
\ No newline at end of file
+- [DevDocs.io](https://devdocs.io) yer imlerinize ekleyiniz. "Hoş Geldiniz" mesajını okuyunuz. Massive API belgeleri koleksiyonu, çevrimdışı da kullanılabiliyor. Kapsanan her şey ve daha fazlası için temel bir başvuru materyali koleksiyonu. (Yöneten: [FreeCodeCamp](https://freecodecamp.org))
```
Comment (hueypov) [inline temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md]:
```suggestion
- [DevDocs.io](https://devdocs.io)'yu yer imlerinize ekleyin. "Hoş Geldin" mesajını okuyun. Çevrimdışı da çalışan devasa API dokümantasyon koleksiyonu. Ele alınan her şey ve daha fazlası için temel bir referans materyal koleksiyonu. (Yöneten: [FreeCodeCamp](https://freecodecamp.org))
```

Comment (CoderMungan) [reply temel_bilgiler/html_css/html-temelleri/html-ve-css-giris.md]: [512643a](https://github.com/kamp-us/turkce-odin-project/pull/54/commits/512643a3b0147e0ea932514271aabed65049c844) done!

**Pull request merged**